pkg/apis/cassandraoperator/v1alpha1/zz_generated.openapi.go

```diff
@@ -1,6 +1,4 @@
-// +build !ignore_autogenerated
-
-// Code generated by openapi-gen. DO NOT EDIT.
+// +build !
```

Comment:
Not sure why the generator made this change. The lonely `!` doesn't make sense to me. This could be a bug in https://github.com/kubernetes/gengo.

Any ideas?

Reply:
Bug reported in https://github.com/operator-framework/operator-sdk/issues/1747.